tensorflow/python/kernel_tests/decode_image_op_test.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for decode_image."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+
+
+class DecodeImageOpTest(tf.test.TestCase):
+
+  def testGif(self):
+    # GIF encoded 2x2 image
+    image_bytes = (b'GIF89a\x02\x00\x02\x00\xf1\x01\x00\x00\x00\x00\xff\xff\xff'
+                   b'\xff\xff\xff\x00\x00\x00!\xf9\x04\x00\x00\x00\x00\x00,\x00'
+                   b'\x00\x00\x00\x02\x00\x02\x00\x00\x02\x03D4\x05\x00;')
```

Comment:
Please use the same test files as `image_ops_test.py`: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/image_ops_test.py#L1953